osm/osm.go

```diff
@@ -34,8 +34,8 @@ import (
 )
 
 const (
-	SecretMountPath = "/etc/osm"
 	CaCertFileName  = "ca.crt"
+	SecretMountPath = "/tmp/osm"
```

Comment:
How does it affect existing databases? @the-redback @iamrz1

Reply:
It is used in snapshot jobs. So, it won't be a problem for existing databases.